docs/templates/query-modules/_calling_query_modules.mdx

```diff
@@ -5,8 +5,16 @@ loaded into Memgraph, you can call them within queries using the following Cyphe
 syntax:
 
 ```cypher
-CALL module.procedure(arg1, "string_argument", ...) YIELD res1, res2, ...;
+CALL module.procedure([optional parameter], arg1, "string_argument", ...) YIELD res1, res2, ...;
 ```
+Every procedure has a first optional parameter and the rest of them depend on the procedure you are trying to call. The optional parameter
+
+must be result of the aggregation function [`project()`](/cypher-manual/functions#aggregation-functions).
+
+If such a parameter is provided, **all** operations will be executed on a projected graph. Otherwise, you will work on
+
+the whole graph stored inside Memgraph. 

```

Comment:
```suggestion
Every procedure has a first optional parameter and the rest of them depend on the procedure you are trying to call. The optional parameter must be result of the aggregation function [`project()`](/cypher-manual/functions#aggregation-functions). If such a parameter is provided, **all** operations will be executed on a projected graph. Otherwise, you will work on the whole graph stored inside Memgraph. 
```